readthedocs/templates/profiles/private/security_log.html

```diff
@@ -0,0 +1,63 @@
+{% extends "profiles/base_profile_edit.html" %}
+
+{% load i18n %}
+{% load pagination_tags %}
+
+{% block title %}{% trans "Security Log" %}{% endblock %}
+
+{% block profile-admin-security-log %}active{% endblock %}
+
+{% block edit_content_header %} {% trans "Security Log" %} {% endblock %}
+
+{% block edit_content %}
+
+<p>
+  {% blocktrans trimmed with days_limit as days_limit %}
+  Showing logs from the last {{ days_limit }} days.
+  {% endblocktrans %}
+</p>
+
+{% autopaginate object_list 15 %}
+
+<div class="module">
+  <div class="module-list">
+    <div class="module-list-wrapper">
+      <ul>
+        {% for log in object_list %}
+        <li class="module-item">
+          {% blocktrans trimmed with ip=log.ip browser=log.browser %}
+          Authenticated from
+          <a href="?ip={{ ip }}" title="{{ browser }}">
+            <code>{{ ip }}</code>
+          </a>
+          {% endblocktrans %}
+          {% if log.log_project_id %}
+            &middot;
```

Comment:
It wasn't clear what this metadata was by looking at it. I would probably move this to the description instead:

`Authentication attempt for project foo from 1.2.3.4`